src/View/Window.vala

```diff
@@ -260,141 +265,148 @@ public class Files.View.Window : Hdy.ApplicationWindow {
 
 
         button_forward.slow_press.connect (() => {
-            get_action_group ("win").activate_action ("forward", new Variant.int32 (1));
+            activate_action ("forward", new Variant.int32 (1));
```

Comment:
This isn't working anymore for some reason

Reply:
The gesture is not triggering - you seem to need to add `propagation_phase = Gtk.PropagationPhase.CAPTURE` to the button gesture properties.  Not sure why it works in main without it, but not here though.